### ISSUE TITLE
WindowsError: [Error 0] The operation completed successfully

### ISSUE THREAD

**Issue by kejxu** [giampaolo/psutil#1410]
psutil 5.5.0 fails while trying to `import gnupg` on Windows 10
Python version: 2.7.15
gnupg version: 2.3.1
psutil version: 5.5.0

```
>pip install psutil --force-reinstall
Collecting psutil
  Using cached https://files.pythonhosted.org/packages/86/96/a7bfcc3aebedd7112ff353204901db6a1a0c1f3555b2788c68842bb78005/psutil-5.5.0-cp27-none-win_amd64.whl
Installing collected packages: psutil
  Found existing installation: psutil 5.4.8
    Uninstalling psutil-5.4.8:
      Successfully uninstalled psutil-5.4.8
Successfully installed psutil-5.5.0
You are using pip version 18.1, however version 19.0.2 is available.
You should consider upgrading via the 'python -m pip install --upgrade pip' command.

>python
Python 2.7.15 (v2.7.15:ca079a3ea3, Apr 30 2018, 16:30:26) [MSC v.1500 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import gnupg
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\opt\python27amd64\lib\site-packages\gnupg\__init__.py", line 23, in <module>
    from . import gnupg
  File "C:\opt\python27amd64\lib\site-packages\gnupg\gnupg.py", line 42, in <module>
    from ._meta    import GPGBase
  File "C:\opt\python27amd64\lib\site-packages\gnupg\_meta.py", line 134, in <module>
    class GPGBase(object):
  File "C:\opt\python27amd64\lib\site-packages\gnupg\_meta.py", line 72, in __new__
    if cls._find_agent():
  File "C:\opt\python27amd64\lib\site-packages\gnupg\_meta.py", line 115, in _find_agent
    if (proc.name() == "gpg-agent") and proc.is_running():
  File "C:\opt\python27amd64\lib\site-packages\psutil\__init__.py", line 609, in name
    name = self._proc.name()
  File "C:\opt\python27amd64\lib\site-packages\psutil\_pswindows.py", line 635, in wrapper
    return fun(self, *args, **kwargs)
  File "C:\opt\python27amd64\lib\site-packages\psutil\_pswindows.py", line 687, in name
    return py2_strencode(os.path.basename(self.exe()))
  File "C:\opt\python27amd64\lib\site-packages\psutil\_pswindows.py", line 635, in wrapper
    return fun(self, *args, **kwargs)
  File "C:\opt\python27amd64\lib\site-packages\psutil\_pswindows.py", line 701, in exe
    return py2_strencode(convert_dos_path(cext.proc_exe(self.pid)))
WindowsError: [Error 0] The operation completed successfully
```

this might be a duplicate with https://github.com/giampaolo/psutil/issues/1405. Thanks!

**Comment by giampaolo** [comment]
I just released 5.5.1 version which fixes this problem. Also closing this out as a duplicate of #1394.

**Comment by kejxu** [comment]
> I just released 5.5.1 version which fixes this problem. Also closing this out as a duplicate of #1394.

thanks for checking this issue and unblocking us with the recent fix!

**Comment by giampaolo** [comment]
No problem. Be good.
